src/api/libcellml/units.h

```diff
@@ -428,41 +428,41 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
     /**
      * @brief Return the scaling factor difference between two @c Units.
      *
-     * This can be interpreted as factor, where units2 = (factor)*units1.  This method
-     * does not check to see if the units are compatible.
+     * This can be interpreted as factor, where units2 = (factor)*units1.  If the units
+     * are not compatible the factor returned is 0.
```

Comment:
`0.0` rather than `0` since we are returning a `double`?

Reply:
Gah! I just changed it back because "0.0." looked weird ...

Reply:
It may look weird, but personally I like it when it's clear, from the documentation, what type we are returning.